content/topics/solo-learn/javascript-intermediate/1-objects/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+content_type: topic
+ready: true
+tags:
+- javascript
+- solo-learn
+title: Solo Learn - JavaScript Intermediate - Objects
```

Comment:
The last module in the intro course should be a prerequisite to this course. They should not start this one once the other one is completed.